university-management-system/src/test/java/excepiton/ProfessorDuplicateExceptionTest.java

```diff
@@ -0,0 +1,32 @@
+package excepiton;
+
+import exception.ProfessorDuplicateException;
+import org.junit.jupiter.api.Test;
+import tables.Faculty;
+import tables.Professor;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class ProfessorDuplicateExceptionTest {
+    @Test
+    void shouldThrowException() {
+        Professor professor = new Professor("Pityu", Faculty.ENGINEERING, 5);
+
+        assertThrows(
+                ProfessorDuplicateException.class, () -> {
+                    throw new ProfessorDuplicateException(professor);
```

Comment:
so you are throwing an exception and asserting that that exception is thrown? :D i think you understand this makes no sense at all